src/CoreBundle/Field/FieldTreeNodePortalSelect.php

```diff
@@ -19,13 +23,37 @@
 class FieldTreeNodePortalSelect extends \TCMSFieldTreeNode
 {
     /**
-     * {@inheritDoc}
+     * @var string
      */
-    public function _GetOpenWindowJS()
+    protected $treeNodeSelectModulePagedef = 'navigationTreeSingleSelect';
+
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
+
+    public function GetHTML()
     {
-        $url = PATH_CMS_CONTROLLER.'?pagedef=navigationTreeSingleSelect&fieldName='.urlencode($this->name).'&id='.urlencode($this->data).'&portalSelect=1';
-        $js = "CreateModalIFrameDialogCloseButton('".\TGlobal::OutHTML($url)."')";
+        $this->translator = $this->getTranslator();
```

Comment:
Is this used somewhere else (otherwise it doesn't need to be an instance variable)?

Reply:
changed to normal variable